tools/setup.py

```diff
@@ -19,7 +18,6 @@ def main():
     third_party.download_gn()
     third_party.download_clang_format()
     third_party.download_clang()
-    prebuilt.load()
```

Comment:
If it's not too much trouble, it would be good to have two commits- one that reverts the prebuilt and another which fixes cargo build.

Reply:
Oops. I see you already did that. Mention which commit you're reverting in the revert description?

Reply:
Look closely at the commits list :)